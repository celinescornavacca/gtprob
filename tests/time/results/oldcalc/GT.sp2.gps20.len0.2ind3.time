0.28user 0.15system 0:00.57elapsed 75%CPU (0avgtext+0avgdata 30392maxresident)k
125inputs+0outputs (0major+12776minor)pagefaults 0swaps
