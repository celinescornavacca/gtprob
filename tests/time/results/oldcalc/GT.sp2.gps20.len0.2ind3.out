-110.216952549
